### DISCUSSION TITLE
How do I change the minimum number of Items that are Enabled

### DISCUSSION THREAD

**Discussion by lvedassp** [LanKuDot/game_modules#39] [Q&A]
![asdasdasd](https://user-images.githubusercontent.com/112566062/210055280-f7b90582-2d6f-4568-84e1-be2782d63070.png)

When I scroll to the extreme of the list, only three buttons are visible, how do I increase or decrease this value?
I want atleast 5 buttons to be always visible on the list.

Thank you.

**Comment by LanKuDot** [comment]
Sorry. In the current version, the list will be stopped at the center.
But in the next version, there will be an option to specify where the list should be stopped at.

**Reply by LanKuDot** [reply]
New version is completed, and now it supports the difference focusing position, which could make the list be stopped at the top or bottom position.
Here is the demonstration: https://youtu.be/6y-_MaeWIAg?t=32
